src/aat/resources/features/F-035 - Get Case V2 External/F-035.feature

```diff
@@ -93,4 +93,112 @@ Scenario: must return status 200 along with the case-view object successfully
       And the response [contains Last State Modified Date metadata field],
       And the response has all other details as expected.
 
-#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+#-- Group Access GA-13 Tests -------------------------------------------------------------------------------------------------
+  @S-960
+  Scenario: User should be able to access a case as role assignment attribute and case caseAccessGroupId both match along with matching AccessProfiles
+    Given a case that has just been created as in [GroupAccess_Full_Case_Creation_Data],
+    And a user with [an active profile in CCD],
+
+    When a request is prepared with appropriate values,
+    And the request [contains the case reference of the case just created],
+    And it is submitted to call the [retrieve a case by id] operation of [CCD Data Store],
+
+    Then a positive response is received,
+    And the response [contains the details of the case just created, along with an HTTP-200 OK]
+
+  @S-961 @Ignore
```

Comment:
As discussed these tests are to be completed at a later date as agreed with Hemanth + Rebecca